share/data-modeling/domain-driven-design/pattern2/README.md

```diff
@@ -0,0 +1,203 @@
+# Pattern 2: Single Domain, Multiple Bounded Contexts, and a Single Uni
+
+**Prerequisite:** Please familiarize yourself with pattern 1 before you practice pattern 2.
+
+
+# Scenario
+
+In Pattern 1 CRM team created a Uni for ‘CustomerAccount’. Let’s say requirements were different. We will remodel uni with pattern 2. The CRM team created another entity to track customer profiles(likes, habits etc.). The Marketing team is interested in the customer profile dataset but doesn't want to build a separate Uni. Marketing will use customer profile data data for customer research but expect the CRM domain to continue to manage it.
+
+
+# Modeling
+
+In this scenario we see two separate domains trying to leverage a single Uni. CRM domain and the Marketing domain. Marketing doesn’t want to own any data and expects the CRM domain to build and own ‘CustomerProfile’ along with the ‘CustomerAccount’. Therefore, the CRM team will have to model two ‘Bounded Context’ in the same Uni. ‘CustomerAccount’ and ‘CustomerProfile’. However, Marketing will only do Read-Only activities on Uni so the CRM team decided to provide a Read-Only copy to Marketing for customer research and analytics. 
+
+
+# Create a Uni
+
+1. Login to Vendia Share [here](https://share.vendia.net/login). 
+
+  Don’t have a login, you can sign up for one [here](https://share.vendia.net/). 
+
+2. Create Uni
+
+  Click the 'Create Universal Application' button on the top-right. Next, use the ‘Create your own’ option. 
+
+3. Fill out details in the Uni Creation Wizard. 
+
+   **Step 1:** Give your Uni a name. Since this Uni will be owned and operated by the CRM team. I will give it the name ‘CRM Uni’. Remember, A Uni can hold more data models for more than one bounded context such as a customer account. 
```

Comment:
Same feedback as in the section above.  Naming collisions will bite here and in subsequent sections.